src/scripts/main.js

```diff
@@ -1,3 +1,16 @@
 'use strict';
 
-// write your code here
+const list = document.querySelectorAll('.population');
+const total = [...list].reduce(
+  (acc, item) => acc + Number(item.innerText.replace(/,/g, '')),
+  0
+);
+
+const average = total / [...list].length;
+
+const totalString = total.toString().replace(/\B(?=(\d{3})+(?!\d))/g, ',');
+const averageString = average.toString().replace(/\B(?=(\d{3})+(?!\d))/g, ',');
```

Comment:
first, use map to split innerText by comma
then use reduce to find the total number

no need for replace method then